harness/determined/_info.py

```diff
@@ -353,6 +359,16 @@ def container_addrs(self) -> List[str]:
         assert self._rendezvous_info is not None
         return self._rendezvous_info.container_addrs
 
+    @property
+    def container_slots(self) -> List[int]:
```

Comment:
Can we rename this?  I don't think the name clearly reflects the meaning.

If I'm on an 4-node, 2-gpu-per-node job, I would have imagined `container_slots` to be `2`, not `[2,2,2,2]`.

Maybe `container_slot_counts`?  My thinking is that since "slots" is naturally something that is already plural on every node, but the "slot_count" can only be one per node, so the plural "slot_counts" naturally sounds more like it should be a list of slot counts per container.